templates/components/group/info_card.html.twig

```diff
@@ -0,0 +1,62 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2024 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+<div class="p-0 group-info-card">
+   <div class="row">
+      <h4 class="card-title mb-1">
+         {% if group['id'] %}
+            <a href="{{ 'Group'|itemtype_form_path(group['id']) }}">{{ group['group_name'] }}</a>
+         {% else %}
+            {{ group['group_name'] }}
+         {% endif %}
+      </h4>
+
+      <div class="text-muted">
+         {% if group['comment']|length > 0 %}
+            <div>
+               <i class="ti ti-notes"></i>
+               {{ group['comment'] }}
+            </div>
+         {% endif %}
+      </div>
+   </div>
+
+   {% if can_edit %}
+      <div class="col">
+         <a class="btn btn-icon btn-sm btn-outline-secondary" href="{{ path('front/preference.php') }}"
```

Comment:
Why is there a link to preferences in a group tooltip ? Am I missing something ?